chime.me.website.json

```diff
@@ -1,30 +1,85 @@
 {
-   "providerId":"chime.me",
-   "providerName":"Chime",
-   "serviceId":"website",
-   "serviceName":"Chime Real Estate Site",
-   "version": 2,
-   "logoUrl":"https://static.chimeroi.com/home_pictures/Chime-Logo.svg",
-   "description":"Chime customers can use their own domain name or subdomain to access their website built on chime",
-   "syncRedirectDomain":"chime.me",
-   "records":[
-      {
-         "type":"A",
-         "host":"@",
-         "pointsTo": "52.52.24.52",
-         "ttl":3600
-      },
-      {
-         "type":"A",
-         "host":"@",
-         "pointsTo": "52.9.101.47",
-         "ttl":3600
-      },
-      {
-         "type":"CNAME",
-         "host":"%cnamehost%",
-         "pointsTo": "@",
-         "ttl":3600
-      }
-   ]
-}
+  "providerId": "chime.me",
+  "providerName": "Lofty",
+  "serviceId": "website",
+  "serviceName": "Lofty Real Estate Site",
+  "version": 3,
+  "logoUrl": "https://static.chimeroi.com/servicetool-temp/2023910/17/crm/lofty-Logo.svg",
+  "description": "Lofty customers can use their own domain name or subdomain to access their website built on lofty",
+  "syncRedirectDomain": "lofty.com",
+  "records": [
+    {
+      "type": "A",
+      "groupId": "website",
+      "host": "@",
+      "pointsTo": "52.52.24.52",
+      "ttl": 3600
+    },
+    {
+      "type": "A",
+      "groupId": "website",
+      "host": "@",
+      "pointsTo": "52.9.101.47",
+      "ttl": 3600
+    },
+    {
+      "type": "CNAME",
+      "groupId": "email",
+      "host": "rkm6vewmvsxmiamzzwjmudzzv3na6wfy._domainkey",
+      "pointsTo": "rkm6vewmvsxmiamzzwjmudzzv3na6wfy.dkim.amazonses.com",
+      "ttl": 3600
+    },
+    {
+      "type": "CNAME",
+      "groupId": "email",
+      "host": "zsb4wstt72sog45rybxq4pze3dxk7b34._domainkey",
+      "pointsTo": "zsb4wstt72sog45rybxq4pze3dxk7b34.dkim.amazonses.com",
+      "ttl": 3600
+    },
+    {
+      "type": "CNAME",
+      "groupId": "email",
+      "host": "cmqy7iaagy7f4r2dt22inmpmfgikyju2._domainkey",
+      "pointsTo": "cmqy7iaagy7f4r2dt22inmpmfgikyju2.dkim.amazonses.com",
+      "ttl": 3600
+    },
+    {
+      "type": "MX",
+      "groupId": "email",
+      "host": "mail",
+      "pointsTo": "feedback-smtp.us-west-2.amazonses.com",
+      "ttl": 3600,
+      "priority": 10
+    },
+    {
+      "type": "MX",
+      "groupId": "email",
+      "host": "@",
+      "pointsTo": "inbound-smtp.us-west-2.amazonaws.com",
+      "ttl": 3600,
+      "priority": 10
+    },
+    {
+      "type": "TXT",
+      "groupId": "email",
+      "host": "_dmarc",
+      "ttl": 3600,
+      "data": "v=DMARC1; p=none",
+      "txtConflictMatchingMode": "ALL"
```

Comment:
As the values are case sensitive, please use `All` as per spec, not "ALL"

Reply:
ok, I will update it.